athena-gcs/src/test/java/com/amazonaws/athena/connectors/gcs/filter/AthenaReadRecordsRequest.java

```diff
@@ -0,0 +1,86 @@
+/*-
+ * #%L
+ * athena-storage-api
+ * %%
+ * Copyright (C) 2019 - 2022 Amazon Web Services
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.amazonaws.athena.connectors.gcs.filter;
+
+import com.amazonaws.athena.connector.lambda.domain.Split;
+import com.amazonaws.athena.connector.lambda.domain.TableName;
+import com.amazonaws.athena.connector.lambda.domain.predicate.Constraints;
+import com.amazonaws.athena.connector.lambda.records.ReadRecordsRequest;
+import com.amazonaws.athena.connector.lambda.security.FederatedIdentity;
+import org.apache.arrow.vector.types.pojo.Schema;
+
+public class AthenaReadRecordsRequest extends ReadRecordsRequest
```

Comment:
I don't see the point of this class, every method is just going to the super class's implementation.

This class should just be removed.

Again, please find other instances where there is just unnecessary code and eliminate it.

Reply:
Removed nad refactored Test cases.